english/sanfran/bikeride/ginger-peachy-keen/drink-more-teas/explosive-euphoria.md

```diff
@@ -0,0 +1,2 @@
+The world is turning n I'm exploding...Hush-hush I think I've had enough tea for now. 
```

Comment:
It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.